scripts/build-shotcut.sh

```diff
@@ -20,10 +20,13 @@ SOURCE_DIR="$INSTALL_DIR/src"
 ACTION_GET_COMPILE_INSTALL=1
 ACTION_GET_ONLY=0
 ACTION_COMPILE_INSTALL=1
+TAR_APP_PACKAGE=1
+KEEP_BUILD_FILES=0
```

Comment:
Suggestion: Consider renaming "KEEP_BUILD_FILES" to "CLEANUP". It is simpler and means the same thing. You will need to invert the logic, obviously.